code/datums/status_effects/xeno_buffs.dm

```diff
@@ -837,7 +837,7 @@
 /datum/status_effect/blessing/warding/on_apply()
 	if(!isxeno(owner))
 		return FALSE
-	armor_modifier = getArmor(37.8 * strength ,8.1 * strength ,13.5 * strength * 2.7 ,8.1 * strength ,0 ,0 ,0 ,0 )//create armor for puppets
+	armor_modifier = getArmor(37.8 * strength ,8.1 * strength ,13.5 * strength ,8.1 * strength ,0 ,0 ,0 ,0 )//create armor for puppets
```

Comment:
also remove all these zeros, why are you adding extra args?

Reply:
removed and slain.